bobtemplates/plone_addon/src/+package.namespace+/+package.name+/testing.py.bob

```diff
@@ -25,6 +25,9 @@ class {{{ package.browserlayer }}}(PloneSandboxLayer):
 {{% endif %}}
 
     def setUpZope(self, app, configurationContext):
+        # Load any other ZCML from your package' dependencies manually here.
```

Comment:
The Plone fixture disables the autoinclude feature here:
https://github.com/plone/plone.app.testing/blob/master/plone/app/testing/layers.py#L134
but loads other ZCML in the following line...

I'll change to your suggested wording.